config/settings.yml

```diff
@@ -14,12 +14,7 @@ RECAPTCHA:
   SITE_KEY: 6Lc6BAAAAAAAAChqRbQZcn_yyyyyyyyyyyyyyyyy
   SECRET_KEY: 6Lc6BAAAAAAAAKN3DRm6VA_xxxxxxxxxxxxxxxxx
 
-home_page_flash_message_html: |-
-  <h1 class="h3 alert-heading">Welcome to the new My Library Account!</h1>
-
-  <p>My Account has been completely re-designed. It shows more information, highlights items that need attention, requires fewer steps, and is easier to use on mobile devices. <a href="https://library.stanford.edu/blogs/stanford-libraries-blog/2019/08/new-my-library-account">&raquo; Find out more</a></p>
-
-  <p class="mb-0">Use the feedback link to let us know what you think!</p>
+home_page_flash_message_html:
```

Comment:
I think this should just be:
```
internal_pages_flash_message_html: |-
```

This way only the html beneath would need to be removed or added the next time we need a message.

Reply:
Copy paste typo...sorry!